docs/guides/multi-tenant.md

```diff
@@ -0,0 +1,299 @@
+---
+title: Multi-Tenant Crossplane
+toc: true
+weight: 240
+indent: true
+---
+
+This guide describes how to use Crossplane effectively in multi-tenant
+environments by utilizing Kubernetes primitives and compatible policy
+enforcement projects in the cloud-native ecosystem.
+
+- [Background](#background)
+    - [Cluster Scoped Managed Resources](#cluster-scoped-managed-resources)
+    - [Namespace Scoped Claims](#namespace-scoped-managed-resources)
```

Comment:
this link is broken, i think the current subsection title would make it `#namespace-scoped-claims`